sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -47,6 +47,7 @@
     private final ClientLogger logger = new ClientLogger(ServiceBusReceiverAsyncClientIntegrationTest.class);
     private final AtomicInteger messagesPending = new AtomicInteger();
 
+    private List<Long> messagesDeferredPending = new ArrayList<>();
```

Comment:
private final

Reply:
I need to initialize it everytime in `beforeTest`.

Reply:
Why do we need an initialization here then? It'll be overwritten.